### PR TITLE
bevo2 configs

### PR DESCRIPTION
**Pull request by jnunyez**
This PR follows the configurations provided in Bevoguide to mimic Day2 configs using ZTP:

- Adds FEC operator and SRIOVFecClusterConfig 
- SRIOVNetworkNodePolicies 

Also adds TGM config in PTP operator.

### CODE REVIEW

File: clusters/ztp-policies/common/group-policies/group-dellr750-vse6.yaml [hunk 1]
```diff
@@ -53,6 +80,81 @@ spec:
...
+      policyName: "sriov-netd-policy-ens5f1"
+      metadata:
+        name: intel-sriov-netd-ens5f1
+      spec:
+        deviceType: netdevice
+        mtu: 9216
+        isRdma: false
+        nicselector:
+          pfNames:
+          - ens5f1#0-1
```
Comment (novacain1) [inline clusters/ztp-policies/common/group-policies/group-dellr750-vse6.yaml:108]:
```suggestion
          - ens5f0#0-1
```

File: clusters/ztp-policies/common/group-policies/group-dellr750-vse6.yaml [hunk 2]
```diff
@@ -53,6 +80,81 @@ spec:
...
+      policyName: "sriov-netd-policy-ens2f0"
+      metadata:
+        name: intel-sriov-netd-ens2f0
+      spec:
+        deviceType: netdevice
+        mtu: 9216
+        isRdma: false
+        nicselector:
+          pfNames:
+          - ens2f0#3-6
```
Comment (novacain1) [inline clusters/ztp-policies/common/group-policies/group-dellr750-vse6.yaml:153]:
```suggestion
          - ens2f0#3-7
```

Comment (novacain1) [reply clusters/ztp-policies/common/group-policies/group-dellr750-vse6.yaml:153]: needs to add up to max VFs.

File: clusters/ztp-policies/common/group-policies/group-dellr750-vse6.yaml (hunk 1 above)
Comment (novacain1) [reply clusters/ztp-policies/common/group-policies/group-dellr750-vse6.yaml:108]: wrong interface.

File: clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "bevo2"
+  namespace: "bevo2"
+spec:
+  baseDomain: "cars2.lab"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "img4.14.10-x86-64-appsub"
```
Comment (novacain1) [inline clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:13]:
```suggestion
  clusterImageSetNameRef: "img4.14.11-x86-64-appsub"
```

File: clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            name: "r750-2-bmc-creds-secret"
+          bootMACAddress: "b4:96:91:a5:04:ac"
+          # Use UEFISecureBoot to enable secure boot
+          bootMode: "UEFI"
+          rootDeviceHints:
+            deviceName: "/dev/disk/by-path/pci-0000:65:00.0-scsi-0:2:0:0"
+          nodeNetwork:
+            interfaces:
+              - name: ens2f0
+                macAddress: "b4:96:91:a5:04:ac"
```
Comment (novacain1) [inline clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:61]:
```suggestion
          nodeNetwork:
            interfaces:
              - name: eno12399np0
                macAddress: "e4:3d:1a:dd:b8:20"
              - name: ens2f0
                macAddress: "b4:96:91:a5:04:ac"
              - name: ens5f0
                macAddress: "50:7c:6f:1f:b3:58"
```

Comment (novacain1) [reply clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:61]: add broadcom interface like bevo1.  also add other connected (with carrier) interfaces.

File: clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+                    - 192.168.38.12
+                    - 2600:52:7:38::12
+              routes:
+                config:
+                  - destination: 0.0.0.0/0
+                    next-hop-address: 192.168.38.129
+                    next-hop-interface: ens2f0
+                  - destination: ::/0
+                    next-hop-address: 2600:52:7:300::1
+                    next-hop-interface: ens2f0
```
Comment (novacain1) [inline clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:92]:
```suggestion
                - name: eno12399np0
                  type: ethernet
                  state: up
                  ipv4:
                    enabled: true
                    address:
                      - ip: "192.168.38.151"
                        prefix-length: 26
                    dhcp: false
                  ipv6:
                    enabled: true
                    address:
                      - ip: "2600:52:7:300::151"
                        prefix-length: 64
                    autoconf: false
                    dhcp: false
                - name: ens2f0
                  type: ethernet
                  state: down
                - name: ens5f0
                  type: ethernet
                  state: down
              dns-resolver:
                config:
                  server:
                    - 192.168.38.12
                    - 2600:52:7:38::12
              routes:
                config:
                  - destination: 0.0.0.0/0
                    next-hop-address: 192.168.38.129
                    next-hop-interface: eno12399np0
                  - destination: ::/0
                    next-hop-address: 2600:52:7:300::1
                    next-hop-interface: eno12399np0
```

File: clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+      nodes:
+        - hostName: "r750-2.bevo2.cars2.lab"
+          role: "master"
+          # Optionally; This can be used to configure desired BIOS setting on a host:
+          # biosConfigRef:
+          # filePath: "example-hw.profile"
+          bmcAddress: "idrac-virtualmedia://192.168.38.204/redfish/v1/Systems/System.Embedded.1"
+          bmcCredentialsName:
+            name: "r750-2-bmc-creds-secret"
+          bootMACAddress: "b4:96:91:a5:04:ac"
```
Comment (novacain1) [inline clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:53]:
```suggestion
          bootMACAddress: "e4:3d:1a:dd:b8:20"
```

Comment (novacain1) [reply clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:53]: switch this to default router interface (eno12399np0).

File: clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml (hunk 5 above)
Comment (novacain1) [reply clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:92]: switch default route to broadcom, unless i'm missing something here?  also you need to add other interfaces and set to `down` so those interfaces don't DHCP constantly (undesired).

**Review by novacain1 — CHANGES_REQUESTED** [review]
See review, some potential problems here.

File: clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml (hunk 4 above)
Comment (jnunyez) [reply clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:61]: Added a few more, and ipaddress confiuration in the broadcom.

File: clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml (hunk 5 above)
Comment (jnunyez) [reply clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml:92]: Not missing anything. Also added all the ports and states.

**Pull request closed without merging**